### DISCUSSION TITLE
Can't add my custom app to search bar (mod+d) using .desktop file

### DISCUSSION THREAD

**Discussion by Pismice** [i3/i3#5777] [Q&A]
I don't know if it's a problem if i just link my stackoverflow issue, if it is a problem just tell me and i can do the whole post again here.
https://stackoverflow.com/questions/77511164/cant-add-my-custom-app-to-i3-search-bar-modd-using-desktop-file
(you obviously can answer here)

**Comment by okraits** [comment]
It looks like `dmenu_run` doesn't consider .desktop files. From the manpage:
`dmenu_run is a script used by dwm which lists programs in the user's $PATH and runs the result in their $SHELL.`

**Reply by Pismice** [reply]
Thank youuuu how did you know that i3 was using dmenu tho ?

**Reply by okraits** [reply]
It's used in i3's default config file.
